docs/css/prism.css

```diff
@@ -6,7 +6,6 @@ pre[class*="language-"] {
   background: 0 0;
   text-shadow: 0 1px #fff;
   font-family: Consolas, Monaco, "Andale Mono", "Ubuntu Mono", monospace;
-  font-size: 1em;
```

Comment:
This made code in paragraph too big:

![image](https://github.com/marmelab/react-admin/assets/99944/74a3a7da-dd4f-4131-b9ac-b1157e556efd)